### DISCUSSION TITLE
то ли баг то ли фича

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#490] [Show and tell]
...

**Comment by Kor-Angar** [comment]
ворота не смог в 5ой главе открыть. ключ не воровался. побил мента у него ключа тоже не оказалось

**Reply by Kor-Angar** [reply]
![20230205222414_1](https://user-images.githubusercontent.com/49514764/216841463-cb4ef0bf-9893-43df-a39d-ccc6c1839470.jpg)
![20230205222349_1](https://user-images.githubusercontent.com/49514764/216841470-9c32216c-6a49-4d9a-9225-42e413d21ce4.jpg)
![20230205222334_1](https://user-images.githubusercontent.com/49514764/216841478-fb713322-3b4b-423a-84c3-8eeadcab5f79.jpg)

**Reply by dosinabox** [reply]
последствия использования старых сохранений

**Reply by Kor-Angar** [reply]
не факт

**Reply by dosinabox** [reply]
надеюсь, текущее прохождение скоро закончится и дальнейшие тесты будут более точными)

**Reply by Kor-Angar** [reply]
я тут понять не могу почему у меня при старте новый игры происходит запуск в минентале 4. даже если я в хориносе главу поменял на пятую :-)

**Reply by dosinabox** [reply]
поподробнее?

**Reply by Kor-Angar** [reply]
поудалял всё что можно. может завтра всё работает. просто при входе в миненталь стартует 4 глава. от чего понять не могу. никаких скрытых настроек про такое нет?

**Reply by Kor-Angar** [reply]
вообщем это было чтение настроек из ini файла :-) https://github.com/dosinabox/g2nr_unofficial_update/discussions/352
